sources.go

```diff
@@ -43,3 +44,14 @@ func Args() *Pipe {
 	}
 	return Echo(s.String())
 }
+
+// Get creates a pipe containing the output of retrieving the web resource
+// given by the supplied url.
+func Get(url string) *Pipe {
+	p := NewPipe()
+	res, err := http.Get(url)
```

Comment:
One thing that concerns me about this is the lack of a timeout in the default HTTP client, for example: https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779

The way I've tackled this in other programs is to have the default behaviour be to use the default client, but allow the user to override this by creating their own client and injecting it somehow. What do you think we should do here?